components/admin_console/team_channel_settings/team/details/team_details.jsx

```diff
@@ -111,6 +118,27 @@ export default class TeamDetails extends React.PureComponent {
         let saveNeeded = false;
```

Comment:
This line would then go below the `if (this.teamToBeArchived())` and `else if (this.teamToBeRestored() && this.state.serverError === null)` blocks, to be a separate variable declaration.